p2p/ipld/nmt_wrapper.go

```diff
@@ -52,11 +57,14 @@ func (w *ErasuredNamespacedMerkleTree) Push(data []byte, idx rsmt2d.SquareIndex)
 		panic("pushed past predetermined square size")
```

Comment:
Optional nit: This panic could also be more helpful by printing the boundary `2*uint(w.squareSize)` and the idx using fmt.Sprintf.

Reply:
good idea https://github.com/lazyledger/lazyledger-core/pull/306/commits/6b7cb81be43706ce33d4330f471a5601eebd3ca6